### PR TITLE
Add CVE-2022-24897 for GHSA-cvx5-m8vg-vxgc

### PR DESCRIPTION
**Pull request by shelbyc**
Add CVE-2022-24897 for GHSA-cvx5-m8vg-vxgc

### CODE REVIEW

File: 2022/24xxx/CVE-2022-24897.json [hunk 1]
```diff
@@ -1,18 +1,101 @@
 {
```
Comment (cve-team) [inline 2022/24xxx/CVE-2022-24897.json:1]: ' https://jira.xwiki.org/browse/XWIKI-5168' does not match '^(ftp|http)s?://\\S+$'

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

File: 2022/24xxx/CVE-2022-24897.json [hunk 2]
```diff
@@ -1,18 +1,101 @@
...
+                "name": "https://github.com/xwiki/xwiki-commons/pull/127",
+                "refsource": "MISC",
+                "url": "https://github.com/xwiki/xwiki-commons/pull/127"
+            },
+            {
+                "name": "https://github.com/xwiki/xwiki-commons/commit/215951cfb0f808d0bf5b1097c9e7d1e503449ab8",
+                "refsource": "MISC",
+                "url": "https://github.com/xwiki/xwiki-commons/commit/215951cfb0f808d0bf5b1097c9e7d1e503449ab8"
+            },
+            {
```
Comment (anthnysingleton) [inline 2022/24xxx/CVE-2022-24897.json:90]: Hey @shelbyc looks like there is a leading white space in this URL object. Fixing that will unblock processing.

**Comment by anthnysingleton** [comment]
Hello @shelbyc,

This PR has been in a blocked state for a few days. I have gone ahead and pulled the file in a manually process the CVE Record. 

https://www.cve.org/CVERecord?id=CVE-2022-24897

I will not close this PR. If you need to make any additional changes to this record please feel free to open a new PR.

Thanks,
CVE Content Team

File: 2022/24xxx/CVE-2022-24897.json (hunk 2 above)
Comment (shelbyc) [reply 2022/24xxx/CVE-2022-24897.json:90]: Thank you for your help!

**Pull request closed without merging**